common-react/@dbeaver/ui-kit/src/Spin/Spin.tsx

```diff
@@ -0,0 +1,21 @@
+import clsx from 'clsx';
+import './Spin.css';
+import type { ControlSize } from '../types/controls.js';
+
+export interface SpinProps {
+  enabled?: boolean;
+  className?: string;
+  size?: ControlSize;
+}
+
+export function Spin({ className, size, enabled = true }: SpinProps) {
```

Comment:
i think that `Spinner` is more common name for such component (I found Spin only in Ant design)